### ISSUE TITLE
Missing final source code

### ISSUE THREAD

**Issue by biloshkurskyi-ss** [appcoda/SwiftUnitTestDemo#1]
Hello,
in the [http://www.appcoda.com/unit-testing-swift/](url) in the end of article you wrote: "For the final project, you can download it from GitHub." I have upload this project but didn't see any finished project, only your start project.

By the way, if use your tutorial, and write line: `vc = storyboard.instantiateInitialViewController() as! ViewController`
Debugger write error: `Use of unresolved identifier 'vc'`

**Comment by dispatchswift** [comment]
@simonng You have to do 'let vc = storyboard.instantiateInitialViewController() as! ViewController'

**Comment by ham118** [comment]
Yes, this demo does not contain a final source code which has mentioned in the blog
